src/components/v2/V2Project/TreasuryStats/ProjectBalance.tsx

```diff
@@ -22,32 +22,36 @@ export default function ProjectBalance({ style }: { style?: CSSProperties }) {
   } = useContext(V2ProjectContext)
 
   return (
-    <StatLine
-      loading={balanceInDistributionLimitCurrencyLoading}
-      statLabel={<Trans>In treasury</Trans>}
-      statLabelTip={
-        <Trans>The balance of this project in the Juicebox contract.</Trans>
-      }
-      statValue={
-        <div
-          style={{
-            ...textPrimary,
-            color: colors.text.brand.primary,
-            marginLeft: 10,
-          }}
-        >
-          {distributionLimitCurrency?.eq(V2_CURRENCY_USD) && (
-            <span style={textSecondary(theme)}>
-              <ETHAmount amount={ETHBalance} precision={4} padEnd={true} />{' '}
-            </span>
-          )}
-          <V2CurrencyAmount
-            amount={balanceInDistributionLimitCurrency ?? BigNumber.from(0)}
-            currency={distributionLimitCurrency}
-          />
-        </div>
-      }
-      style={style}
-    />
+    <>
```

Comment:
**suggestion**: I think we can remove the wrapping fragment here.

Reply:
AH thanks yep - moved some stuff around and forgot to change back, good pickup